### PR TITLE
finished express-post-json

### PR DESCRIPTION
**Pull request by byu014**
expresspost

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  if (!name || !course || !score) {
+    res.status(400).send('Missing argument(s)');
+    return;
+  }
+  grades[nextId] = { id: nextId, name, course, score };
+  res.status(201).send(grades[nextId++]);
+});
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
```
Comment (scbowler) [inline express-post-json/index.js:19]: You were supposed to send back an array of grades like in the previous exercise

**Review by scbowler — CHANGES_REQUESTED** [review]
POST looks great! GET needs updated. Let me know if you have any questions!

File: express-post-json/index.js (hunk 1 above)
Comment (byu014) [reply express-post-json/index.js]: sorry I didn't catch that I updated it to respond with an array now

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**